contentcuration/contentcuration/frontend/administration/views/UserActionsBulk.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
Could you rename this to UserTableActions? My initial thought was that this is referring to vuex